A fraction part of datetime/timestamp value is truncated

## Summary
- The `datetime`/`timestamp` value can include a trailing fractional seconds part. (like this: **YYYY-MM-DD hh:mm:ss[.fraction]**)
- `deno_mysql` currently truncates a fraction part of these types.

## Steps to reproduce

```typescript
import { Client, Connection } from "https://deno.land/x/mysql@2.2.0/mod.ts";
import { assertEquals } from "https://deno.land/std@0.60.0/testing/asserts.ts";

const client = await new Client().connect({
  hostname: "localhost",
  port: 3306,
  username: "test",
  db: "test",
  password: "test",
});

await client.query(`DROP TABLE IF EXISTS test`);
await client.query(`CREATE TABLE test (
  id int(11) NOT NULL AUTO_INCREMENT,
  datetime datetime(6) NOT NULL,
  PRIMARY KEY (id)
) ENGINE=MEMORY DEFAULT CHARSET=utf8mb4`);

const datetime = new Date();
await client.execute(`
  INSERT INTO test (datetime)
  VALUES (?)`, [datetime]);

const expected = datetime;
const [{ datetime: actual }] = await client.query(`SELECT datetime FROM test`);
assertEquals(expected.toISOString(), actual.toISOString()); // See https://github.com/denoland/deno/issues/6643 for why not simply using `assertEquals(actual, expected)`.
```

**Expected behavior**:

The above code shouldn't fail.

**Actual behavior**:

The above code fails with the fowlloing logs:

```shell
INFO connecting localhost:3306
INFO connected to localhost
error: Uncaught AssertionError: Values are not equal:


    [Diff] Actual / Expected


-   "2020-07-07T19:32:24.464Z"
+   "2020-07-07T19:32:24.000Z"

  throw new AssertionError(message);
        ^
    at assertEquals (asserts.ts:183:9)
    at file:///home/uki00a/work/test.ts:22:1
```

## Environment
- deno v1.1.3
- deno_mysql v2.2.0
- MySQL v5.7

## Ref
https://dev.mysql.com/doc/refman/5.6/en/datetime.html

--- a/deps.ts
+++ b/deps.ts
@@ -11,5 +11,5 @@ export {
 export { format as byteFormat } from "https://deno.land/x/bytes_formater@1.2.0/mod.ts";
 export { Hash } from "https://deno.land/x/checksum@1.4.0/mod.ts";
 export { sha256 } from "https://deno.land/x/sha256@v1.0.2/mod.ts";
-export { replaceParams } from "https://deno.land/x/sql_builder@1.5.0/util.ts";
+export { replaceParams } from "https://deno.land/x/sql_builder@v1.6.0/util.ts";
 export * as log from "https://deno.land/x/std@0.53.0/log/mod.ts";

--- a/test.ts
+++ b/test.ts
@@ -119,6 +119,25 @@ testWithClient(async function testQueryDecimal(client) {
   assertEquals(result, [{ decimal_column: value }]);
 });
 
+testWithClient(async function testQueryDatetime(client) {
+  await client.query(`DROP TABLE IF EXISTS test_datetime`);
+  await client.query(`CREATE TABLE test_datetime (
+    id int(11) NOT NULL AUTO_INCREMENT,
+    datetime datetime(6) NOT NULL,
+    PRIMARY KEY (id)
+  ) ENGINE=MEMORY DEFAULT CHARSET=utf8mb4`);
+  const datetime = new Date();
+  await client.execute(
+    `
+    INSERT INTO test_datetime (datetime)
+    VALUES (?)`,
+    [datetime],
+  );
+
+  const [row] = await client.query("SELECT datetime FROM test_datetime");
+  assertEquals(row.datetime.toISOString(), datetime.toISOString()); // See https://github.com/denoland/deno/issues/6643
+});
+
 testWithClient(async function testDelete(client) {
   let result = await client.execute(`delete from users where ?? = ?`, [
     "id",

